### PR TITLE
[stable28] Fix/comment deleting with activities installed

### PR DESCRIPTION
**Pull request by GretaD**
backport of https://github.com/nextcloud/server/pull/45848

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Conflicts

**Review by miaulalala — APPROVED** [review]

File: apps/comments/src/services/CommentsInstance.js [hunk 1]
```diff
@@ -1,33 +1,18 @@
...
+import { t, n } from '@nextcloud/l10n'
+import { PiniaVuePlugin, createPinia } from 'pinia'
 import Vue from 'vue'
 import CommentsApp from '../views/Comments.vue'
 import logger from '../logger.js'
 
+Vue.use(PiniaVuePlugin)
 // eslint-disable-next-line camelcase
-__webpack_nonce__ = btoa(getRequestToken())
+__webpack_nonce__ = getCSPNonce()
```
Comment (skjnldsv) [inline apps/comments/src/services/CommentsInstance.js:15]: `@nextcloud/auth`'s stable28 version doesn't have `getCSPNonce` available.
Revert this line and remove the import
```suggestion
__webpack_nonce__ = btoa(getRequestToken())
```

File: apps/comments/src/services/CommentsInstance.js [hunk 2]
```diff
@@ -1,33 +1,18 @@
...
- * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- *
+ * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import { translate as t, translatePlural as n } from '@nextcloud/l10n'
-import { getRequestToken } from '@nextcloud/auth'
+import { getCSPNonce } from '@nextcloud/auth'
```
Comment (skjnldsv) [inline apps/comments/src/services/CommentsInstance.js:6]:
```suggestion
import { getRequestToken } from '@nextcloud/auth'
```

**Pull request merged**